src/sentry/static/sentry/app/views/settings/organizationSecurityAndPrivacy/organizationSecurityAndPrivacyContent.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router/lib/Router';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import JsonForm from 'app/views/settings/components/forms/jsonForm';
+import Form from 'app/views/settings/components/forms/form';
+import AsyncView from 'app/views/asyncView';
+import {Organization} from 'app/types';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {updateOrganization} from 'app/actionCreators/organizations';
+import organizationSecurityAndPrivacy from 'app/data/forms/organizationSecurityAndPrivacy';
+
+export type OrganizationSecurityAndPrivacyContentProps = {
+  organization: Organization;
+  params: {
+    orgId: string;
+    projectId: string;
+  };
+} & RouteComponentProps<{}, {}>;
+
+class OrganizationSecurityAndPrivacyContent extends AsyncView<
+  OrganizationSecurityAndPrivacyContentProps
+> {
+  static contextTypes = {
+    organization: PropTypes.object.isRequired,
+    router: PropTypes.object,
+  };
+
+  getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
+    const {orgId} = this.props.params;
+    return [['authProvider', `/organizations/${orgId}/auth-provider/`]];
+  }
+
+  handleSave = (data: Organization) => {
+    // This will update OrganizationStore (as well as OrganizationsStore
+    // which is slightly incorrect because it has summaries vs a detailed org)
+    updateOrganization(data);
+  };
+
+  renderBody() {
+    const {organization} = this.context;
+    const {orgId} = this.props.params;
+    const {authProvider} = this.state;
+    const initialData = this.props.organization;
+    const endpoint = `/organizations/${orgId}/`;
+    const access = new Set(organization.access);
+    const features = new Set(organization.features);
+
+    return (
+      <React.Fragment>
+        <SettingsPageHeader title={t('Security & Privacy')} />
+        <Form
+          className="organization-settings-security-and-privacy"
```

Comment:
We use `data-test-id` attribute instead of class names for query selectors that are used strictly in testing

```suggestion
          data-test-id="organization-settings-security-and-privacy"
```